waiter/test-files/jwt/jwks.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Can this be deleted?  Can we generate keys dynamically for our unit tests?  How are you generating these keys?